### PR TITLE
apply pyupgrade

### PR DESCRIPTION
**Pull request by Borda**
### Description

Update Python notation to be better suitable to python 3.6+
in fact, I have just used the `pyupgrade` nook in pre-commit

### Status
**Ready/Work in progress/Hold**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/_version.py [hunk 1]
```diff
@@ -81,7 +80,7 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False,
                                  stderr=(subprocess.PIPE if hide_stderr
                                          else None))
             break
-        except EnvironmentError:
+        except OSError:
             e = sys.exc_info()[1]
```
Comment (wyli) [inline monai/_version.py:84]: This is an automatically generated file, could you revert?

**Review by wyli — COMMENTED** [review]
I guess the goal is to do this kind of commit automatically in the future, Not sure whether it is compatible with the other code formatting tools. But the current changes look good. We can experiment it with the development branch for a while first

File: monai/_version.py (hunk 1 above)
Comment (Borda) [reply monai/_version.py]: reverted, and added to exluded files

**Comment by Borda** [comment]
> not sure whether it is compatible with the other code formatting tools

it shall be and another case it shall be caught by your CI :]

**Review by wyli — APPROVED** [review]
It looks good to me, thanks!

**Pull request merged**